runtime/integration-tests/src/generic/cases/lp/pool_management.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use cfg_types::tokens::CurrencyId;
+use ethabi::{ethereum_types::H160, Token, Uint};
+use frame_support::{
+	assert_ok,
+	traits::{OriginTrait, PalletInfo},
+};
+use frame_system::pallet_prelude::OriginFor;
+use pallet_liquidity_pools::GeneralCurrencyIndexOf;
+
+use crate::{
+	generic::{
+		cases::lp::process_outbound,
+		config::Runtime,
+		env::{Env, EvmEnv},
+	},
+	utils::accounts::Keyring,
+};
+
+#[test]
+fn _test() {
+	add_currency::<centrifuge_runtime::Runtime>()
+}
```

Comment:
I guess this can be removed because it's already tested by `test_for_runtimes` line

Reply:
Just for my internal testing, to easily call a test method. will be removed.